forge-gui/res/cardsfolder/c/creeping_chill.txt

```diff
@@ -1,7 +1,7 @@
 Name:Creeping Chill
 ManaCost:3 B
 Types:Sorcery
-A:SP$ DamageAll | StackDescription$ CARDNAME deals 3 damage to each opponent and | Cost$ 3 B | ValidPlayers$ Player.Opponent | NumDmg$ 3 | SubAbility$ DBGainLife | SpellDescription$ CARDNAME deals 3 damage to each opponent and you gain 3 life.
+A:SP$ DamageAll | ValidPlayers$ Player.Opponent | NumDmg$ 3 | SubAbility$ DBGainLife | StackDescription$ SpellDescription | SpellDescription$ CARDNAME deals 3 damage to each opponent and you gain 3 life.
```

Comment:
I wouldn't say it was redundant:
![image](https://github.com/Card-Forge/forge/assets/8506892/877cfcf5-6547-48f9-80fc-63731f237372)

Reply:
I hope this solution is adequate. Turns out trailing into automatic text by ` and | ` is fairly rare: only two scripts do it.

Reply:
I guess that would work
However I still don't understand why you'd change it at all since it was already fine? 🤔

Reply:
Oh, at the time I was quickly scanning the prospective 4000+ hit list for the bulk of the search and replace to check for anything amiss. Various instances of parameters out of the customary order kept showing up. I guess, in my momentary annoyance, I didn't read too closely when dealing with Creeping Chill. 
I'll do my best to do better on that count.

Reply:
sure, no problem